code/game/world.dm

```diff
@@ -122,6 +122,11 @@ var/global/world_topic_last = world.timeofday
 	* Public Topic Calls
 	* The following topic calls are available without a comms secret.
 	* * * * * * * */
+	//HEARTH edit start
+	#ifdef UTILITY_TGSV4
```

Comment:
It will don't work. If you think what modpack allows it - no.

Reply:
Will be replaced later, dependent on the topic commands Nebula PR.